client/mainmenu/CMainMenu.cpp

```diff
@@ -466,22 +468,37 @@ void CMultiMode::hostTCP()
 {
 	auto savedScreenType = screenType;
 	close();
-	GH.windows().createAndPushWindow<CMultiPlayers>(getPlayerName(), savedScreenType, true, ELoadMode::MULTI);
+	std::vector<std::string> playerNames;
+	getPlayersNames(playerNames);
+	GH.windows().createAndPushWindow<CMultiPlayers>(playerNames, savedScreenType, true, ELoadMode::MULTI);
 }
 
 void CMultiMode::joinTCP()
 {
 	auto savedScreenType = screenType;
 	close();
-	GH.windows().createAndPushWindow<CMultiPlayers>(getPlayerName(), savedScreenType, false, ELoadMode::MULTI);
+	std::vector<std::string> playerNames;
+	getPlayersNames(playerNames);
+	GH.windows().createAndPushWindow<CMultiPlayers>(playerNames, savedScreenType, false, ELoadMode::MULTI);
 }
 
-std::string CMultiMode::getPlayerName()
+void CMultiMode::getPlayersNames(std::vector<std::string> & playerNames)
```

Comment:
Why pass vector as parameter when it can be returned?
```suggestion
std::vector<std::string> CMultiMode::getPlayersNames()
```